updating/migrating-to-multi-payload.adoc

```diff
@@ -0,0 +1,18 @@
+:_content-type: ASSEMBLY
+[id="migrating-clusters-to-multi-payload"]
+= Migrating a cluster to a heterogenous payload
+include::_attributes/common-attributes.adoc[]
+:context: updating-clusters-overview
+
+toc::[]
+
+You can migrate your current homogeneous cluster to a heterogeneous cluster by updating to a heterogeneous manifest-listed payload, allowing you to add mixed architecture workers for future use.
+
+include::modules/migrating-to-multi-arch-cli.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
```

Comment:
I also think it would be a good idea just to reference the multi-arch docs here. 

https://github.com/openshift/openshift-docs/blob/main/post_installation_configuration/multi-architecture-configuration.adoc